pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -482,6 +484,7 @@ stdenv.mkDerivation (finalAttrs: {
     (enableFeature withGme "libgme")
     (enableFeature withGnutls "gnutls")
     (enableFeature withGsm "libgsm")
+    (optionalString (lib.versionAtLeast finalAttrs.version "6.1") (enableFeature withHarfbuzz "libharfbuzz"))
```

Comment:
Please use this pattern @jopejoe1 suggested; in line with the rest of the optional enable flags. They recently cleaned this up.